### ISSUE TITLE
Display inaccuracies

### ISSUE THREAD

**Issue by Zmej75** [AkshaySG14/haystack-public#24]
![2024-09-09_13-24-03](https://github.com/user-attachments/assets/5d69b370-5934-4d38-a06c-67068eb23567)
![2024-09-09_13-22-59](https://github.com/user-attachments/assets/5b141a1b-c430-4cc1-a8c7-5c9b934c32c0)
![2024-09-09_13-21-43](https://github.com/user-attachments/assets/4874f1a2-16f6-4131-8dbc-5e7c7cae1454)
There are overlays when displaying interface elements

**Comment by AkshaySG14** [comment]
Will look into this. For the second issue, is that codeium UI?

**Comment by Zmej75** [comment]
Yes, this is Codeiun Code Lens. But I suppose there will be a problem with any Code lens.

**Comment by AkshaySG14** [comment]
The first part should be fixed in the latest version on the website: 0.17! Please let me know if it is not.

**Comment by Zmej75** [comment]
> The first part should be fixed in the latest version on the website: 0.17! Please let me know if it is not.

Resizing the Primary Side Bar is available when the Menu Bar is active
![image](https://github.com/user-attachments/assets/05f47bd4-d5cc-4b7e-9591-e7330959dda8)
